pkg/kubelet/config/http.go

```diff
@@ -206,5 +206,9 @@ func applyDefaults(pod *api.Pod, url string) error {
 	// Always overrides the namespace.
 	pod.Namespace = kubelet.NamespaceDefault
 	glog.V(5).Infof("Using namespace %q for pod %q from URL %s", pod.Namespace, pod.Name, url)
+
+	// TODO(dchen1107): Remove when Kubelet uses pods directly.
```

Comment:
Yeah I didn't know how to better express the need to do this better. Removed this TODO and on the other file.